src/classes/FORM_Element.cls

```diff
@@ -44,16 +44,12 @@ public with sharing class FORM_Element {
     @AuraEnabled public String[] dataImportFieldMappingDevNames;
 
     /* PS Additional props for FormFields */
+    @AuraEnabled public String id;
     @AuraEnabled public String label;
     @AuraEnabled public Boolean required;
-    @AuraEnabled public String value;
```

Comment:
@vmartinov-sforg - Just wanted to call this change out. Data that used to go into this "value" attribute is now in componentName or dataImportFieldMappingDevNames.